src/components/ec-table/ec-table.story.js

```diff
@@ -37,12 +37,21 @@ const data = [
   ],
 ];
 
+const tooltipConfig = {
+  content: 'This is the tooltip info',
+  classes: ['ec-tooltip--bg-bright', 'ec-tooltip--shadow'],
```

Comment:
I'm not very happy to see these `ec-tooltip` classes here because the ec-tooltip can change to something else and this one will not match it anymore. I'd rather see those as props, e.g.
```
background: 'bright',
hasShadow: true,
```

I'm not sure if it's going to possible to refactor the ec-tooltip to support it because it uses third party lib.

Reply:
I don't know how I can do this, if anyone has time to help me, just ping me

Reply:
I'm also sceptical that this is easily doable without rewriting the entire ec-tooltip. I would merge it without that and create a spike for it.